tests/lock.cpp

```diff
@@ -30,7 +30,7 @@ constexpr std::size_t size = 1<<20;
 constexpr std::size_t cache_size = size;
 
 /** @brief number of threads to spawn for some of the tests */
-constexpr int nThreads = 16;
+constexpr int kThreads = 16;
 /** @brief number of itereations to run for some of the tests */
 constexpr int iter = 10000;
```

Comment:
I fail to understand what the difference between L33 and L35 is?
It seems to me that if one adopts the `kMixedCase` convention, should this not apply to both?